### ISSUE TITLE
{@image_field_name._src} or {@image_field_name._img} does not work within an [if][/if] in Pods template iff the [if] is testing for the same image that is referenced inside.

### ISSUE THREAD

**Issue by regnskygge** [pods-framework/pods#4034]
Here is an example Pods template:
```
<div class="simplest-issue-example">
	[if slide_title_image]
	<img src="{@slide_title_image._src}">
	[/if]		
</div>
```

This results in a DOM element as rendered in Chrome 56:
```
<div class="simplest-issue-example">
	<img src="">		
</div>
```

- If I place the magic tag outside the [if] then it renders as expected. 
- If the image field that is tested in the [if] is different than the image field called within then it works as expected.
- All image fields in this example are single file fields and not multiple.

If there is a way of referencing an image within the context of an [if] that is similar to how it works with [each] then I am missing it.

**Comment by jimtrue** [comment]
This should be corrected now with #4408 @regnskygge Can you verify?

**Comment by jimtrue** [comment]
I can verify this works now as of Pods 2.7RC1
